_data/packages.yml

```diff
@@ -2,51 +2,75 @@
   description: 'Jointly: A Python package for synchronizing multiple sensors with accelerometer data'
   maintainer: ["Arne Herdick", "Felix Musmann", "Ariane Sasso", "Justin Albert", "Bert Arnrich"]
   link: "https://github.com/hpi-dhc/jointly"
+  date-accepted:
+  highlight:
```

Comment:
this allows us to highlight packages that we want on the home page while we are figuring out how to collect additional info.